bigbluebutton-html5/private/config/settings.yml

```diff
@@ -267,8 +267,6 @@ public:
     # Controls whether ICE candidates should be signaled to bbb-webrtc-sfu.
     # Enable this if you want to use Kurento as the media server.
     signalCandidates: false
-    # Forces relay usage only on Firefox. Applies to listen only, webcams and screenshare.
-    forceRelayOnFirefox: false
```

Comment:
Please keep the setting in settings.yml to allow folks to turn this off if they want to.

You can flip that to true (because that seems to be the overall goal of this PR), but it needs to be configurable (-- different bridges have different requirements, folks might want to just not use this if their setup is either controlled or have no meaningful impact, ...)

Reply:
Okay. Do you think it would make sense to have tree options for `forceRelayOnFirefox`:

1. `true`: always use turn server on firefox
2. `false`: do not enforce turn usage
3. `'if_turn_server_available'`: force firefox to use a turn server if there is one available. Use this one as default shipped with BBB

The difference between option 1 and 3 is that in option 1 BBB would not work at all in Firefox if no turn server is configured.

Reply:
@schrd Third one works.
Alternatively: if we want to avoid weirdly-typed configurations mixing bool/strings, you could change the flag's name to indicate that it's behavior is lenient (ie swap `forceRelayOnFirefox` for `tryRelayOnFirefox`) and make it behave like option 3. IMO this would be preferable, but whatever seems easier to deal with/document/use.

Reply:
`forceRelayOnFirefox=true` could just be ignored if there are no TURNs configured.